app/src/main/java/de/westnordost/streetcomplete/quests/cycleway/AddCycleway.kt

```diff
@@ -293,7 +302,7 @@ class AddCycleway : OsmElementQuestType<CyclewayAnswer> {
                 or highway = residential and (
                   maxspeed > 30
```

Comment:
It might be good to have a comment that these are in kph, I was a bit confused what was going on at first.

Reply:
nah, that's the default in OSM. The filter syntax just looks up what are inthe OSM tags, and there is something like "30" or "50" etc. in the OSM tags. (Actually, `>` comparisons don't work with "30 mph" because that is not a number but a string). Hence the elaborate regex syntax you also commented on.